website/next.config.js

```diff
@@ -20,16 +22,20 @@ const withNextra = nextra({
   codeHighlight: false,
   defaultShowCopyCode: false,
   transform(result, { route }) {
```

Comment:
it's not a bug, because the route is parsed based on filename of md/mdx file

example:
```
/en/my-page.mdx -> /en/my-page
/en/my-folder/index.mdx -> /en/my-folder
/[locale]/index.mdx -> can't parse, route: ''
```

so `buildGetStaticProps` helper function we can use everywhere except dynamic routes aka `[locale]/index.mdx`

Reply:
Thank you @dimaMachina 🙏